src/controllers/orderController.js

```diff
@@ -27,6 +27,11 @@ const createOrder = async (req, res) => {
     amount: total * 100, // Stripe requires amount in cents
     currency: 'usd',
     description: `Order #${order._id}`,
+    metadata: {
+      userId: req.user.userId,
+      orderId: order._id, // key-values
+      totalQuantity: orderItems.length, // the total number of albums purchased in order
```

Comment:
If we have orderItems like:

```javascript
[
 { album: ..., quantity: 3 }
 { album: ..., quantity: 2 }
]
```

The user has bought 2 albums, but a total quantity of items of 5. 

Maybe we can include both values here in the metadata in case they are of use later.

Reply:
<img width="602" alt="screen4" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/ef297b26-9f39-4831-9577-331d822e66a7">
console.log("Total Quantity:", orderItems.reduce((total, item) => total + item.quantity, 0));
console.log("Total Albums:", orderItems.length);  
I trying to check on console.log, and test in postman, i think its work.